packages/@webex/webex-core/src/lib/services/services.js

```diff
@@ -752,6 +758,30 @@ const Services = WebexPlugin.extend({
     return catalog.findServiceFromClusterId(params);
   },
 
+  /**
+   * @param {String} cluster the cluster containing the id
+   * @param {UUID} [id] the id of the conversation.
+   *  If empty, just return the base URL.
+   * @returns {String} url of the conversation
+   */
+  getUrlFromClusterId({cluster = 'us', id} = {}) {
+    let clusterId = cluster === 'us' ? DEFAULT_CLUSTER_IDENTIFIER : cluster;
+
+    // Determine if cluster has service name (non-US clusters from hydra do not)
+    if (clusterId.split(':').length < 4) {
+      // Add Service to cluster identifier
+      clusterId = `${cluster}:${CLUSTER_SERVICE}`;
+    }
+
+    const {url} = this.getServiceFromClusterId({clusterId}) || {};
+
+    if (!url) {
+      throw Error(`Could not find service for cluster [${cluster}]`);
+    }
+
+    return id ? `${url}/conversations/${id}` : url;
```

Comment:
just return the url here and make sure the conversation also consumes the same function but only appends the id in the end